numcodecs/zfpy.py

```diff
@@ -11,19 +10,8 @@
     _zfpy_version = tuple(map(int, version("zfpy").split(".")))
 
 if _zfpy_version:
-    # Check NumPy version
-    _numpy_version: tuple = tuple(map(int, version("numpy").split('.')))
-    if _numpy_version >= (2, 0, 0) and _zfpy_version <= (1, 0, 1):  # pragma: no cover
```

Comment:
Here, I think we can keep all the code as is, but just fix it to be `_zfpy_version <= (1, 0, 0)` instead.

Reply:
Yes. That makes sense. 
Will change to 
if _numpy_version >= (2, 0, 0) and _zfpy_version < (1, 0, 1):